src/controller.cpp

```diff
@@ -16,17 +17,20 @@ void Controller::configure(
   costmap_ros_ = costmap_ros;
   tf_buffer_ = tf;
   name_ = name;
+  parameters_handler_ = std::make_unique<ParametersHandler>(parent);
```

Comment:
Should be `parameters_handler_.reset()` in `on_cleanup` to be complimentary to its creation in `configure`

Reply:
don